e2e/nomostest/testpredicates/predicates.go

```diff
@@ -430,7 +432,7 @@ func DeploymentMissingEnvVar(containerName, key string) Predicate {
 // IsManagedBy checks that the object is managed by configsync, has the expected
 // resource manager, and has a valid resource-id.
 // Use diff.IsManager if you just need a boolean without errors.
-func IsManagedBy(nt *NT, scope declared.Scope, syncName string) Predicate {
+func IsManagedBy(scheme *runtime.Scheme, scope declared.Scope, syncName string) Predicate {
```

Comment:
nit: in other package funcs that take the scheme, the scheme is the last arg, like kinds.Lookup. But I don't know if it really matters. It's kind of like a Context, which is always the first arg...

Reply:
This sounds tedious to change but if it matters I can switch the argument ordering. I'm not familiar with that convention so let me know how important this is